docs/how-to/use-native-token.md

```diff
@@ -0,0 +1,273 @@
+---
+comments: true
+---
+
+This document show you how you to set up and test a native token in a CDK stack. 
+
+## Summary
+
+!!! quote "From the [Unified bridge documentation](https://github.com/0xPolygonHermez/zkevm-techdocs/blob/a6d46da98ad32ace544e5dbc31d34831f9cc1bdd/knowledge-layer/architecture/PDFs/ulxly.pdf)"
+    When a token is utilized to cover gas expenses within a layer, we refer to it as the gas token for that specific layer.
+
+    Even when employing a gas token within a layer, it remains feasible to transfer L1 ETH to that layer. In such instances, the ETH is recorded within an ERC20 contract known as W-ETH, functioning as another instance of an ERC20 token.
+
+The diagram below illustrates the interchange of assets between layers, focusing on LY as a layer of interest. 
+
+It depicts several scenarios, such as bridging an ERC20 token from mainnet to another ERC20 token in LY, bridging L1 ETH to the LY gas token, or bridging a wrapped ERC20 token living on LX to LY ETH.
+
+![Unfied bridge exchanges](../img/how-to/gas-token-img/ulxly-exchanges.drawio.png)
+
+## Set up and prerequisites
+
+1. Edit the [params.yml](https://github.com/0xPolygon/kurtosis-cdk/blob/main/params.yml) file to set `zkevm_use_gas_token_contract` to `true`. This forces a couple of changes at setup time:
+
+    1. During contract deployment, an ERC20 token is deployed on L1.
+    2. The [rollup creation parameters](https://github.com/0xPolygon/kurtosis-cdk/blob/main/templates/contract-deploy/create_rollup_parameters.json) file is modified to set the `gasTokenAddress` to the address of the L1 ERC20.
+
+2. You can now run the stack:
+
+    ```sh
+    kurtosis run --enclave cdk-v1 --args-file params.yml --image-download always .
+    ```
+
+    !!! tip
+        For full set up and deploy instructions, check out the [quickstart](../quickstart/deploy-stack.md) documentation.
+
+    It takes a few minutes to compile and deploy the full set of contracts. 
+    
+    The screenshot below shows the full set of deployed services and highlights the bridge UI, L1 RPC, and L2 RPC services which we will focus on throughout this document.
+
+    ![Deployed services](../img/how-to/gas-token-img/services.png)
+
+### Mint L1 token
+
+!!! tip
+    For this section you will need to have [Foundry](https://book.getfoundry.sh/getting-started/installation), [jq](https://jqlang.github.io/jq/), and [yq](https://github.com/mikefarah/yq/#install) installed.
+
+1. Extract the `gasTokenAddress` from the generated files:
+
+    ```sh
+    kurtosis service exec cdk-v1 contracts-001 "cat /opt/zkevm/create_rollup_parameters.json"
+    ```
+
+2. Run the following command to mint some L1 tokens.
+
+    ```sh
+    export gta=$(kurtosis service exec cdk-v1 contracts-001 "cat /opt/zkevm/create_rollup_parameters.json" | tail -n +2 | jq -r .gasTokenAddress)
+    export l1_rpc_url=http://$(kurtosis port print cdk-v1 el-1-geth-lighthouse rpc)
```

Comment:
We updated the `ethereum-package` and now `kurtosis port print cdk-v1 el-1-geth-lighthouse rpc` includes the `http` schema.

```bash
$ kurtosis port print cdk-v1 el-1-geth-lighthouse rpc
http://127.0.0.1:55077
```

```suggestion
    export l1_rpc_url=$(kurtosis port print cdk-v1 el-1-geth-lighthouse rpc)
```